www/_template/guides/svelte.md

```diff
@@ -1,12 +1,294 @@
+
+```bash
+cd svelte-snowpack
+npm run start
+```
+
+You should see your new website up and running!
+
+> 💡 Tip: the `README.md` in your new project contains useful information about what each file does.
+
+<div class="frame"><img src="/img/guides/react/minimalist-hello-world.png" alt="screenshot of project-template-minimal, which shows 'Hello world' in text on a white background." class="screenshot"/></div>
+
+Now that you have a basic project up and running, to install Svelte, run the following command in your project directory:
+
+```bash
+npm install svelte --save
+```
+
+> 💡 Tip: add the "--use-yarn" or "--use-pnpm" flag to use something other than npm
+
+You'll also need the `@snowpack/plugin-svelte` plugin so that Snowpack knows how built `.svelte` files into JavaScript and CSS files that run in the browser. Snowpack plugins are a way to extend Snowpack's capabilities without having to do custom configuration. To start, install the package in your project:
+
+```bash
+npm install @snowpack/plugin-svelte --save-dev
+```
+
+Once installed, you'll need to add the plugin to your Snowpack configuration file so that Snowpack knows to use it:
+
+```diff
+  module.exports = {
+    mount: {
+      public: '/',
+      src: '/_dist_',
+    },
+-   plugins: []
++   plugins: ['@snowpack/plugin-svelte'],
+  };
+```
+
+`@snowpack/plugin-svelte` also has built in HMR (hot module replacement) and Fast Refresh, features you'll explore in the next step.
+
+## Create your first Svelte component
+
+Svelte relies on HTML-based templating in .svelte files. Create your first `.svelte` component named `App.svelte` with the following code
+
+```html
+<script>
+  /* Svelte component logic will go here! */
+</script>
+<style>
+  /* Styling will go here! */
+</style>
+<div class="App">
+  <header class="App-header">
+    <a
+      class="App-link"
+      href="https://svelte.dev"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      Learn Svelte
+    </a>
+  </header>
+</div>
+```
+
+Now head to `index.js` and import it the new `app.svelte` file
+
+```diff
+/* Add JavaScript code here! */
+-console.log('Hello World! You did! Welcome to Snowpack :D');
++import App from "./App.svelte";
+
++var app = new App({
++  target: document.body,
++});
+
++export default app;
+```
+
+Start up Snowpack with `npm start` and you should see a page that says "Learn Svelte." GrewR job, you now have Svelte up and running. When you make changes to the code and save them, you'll see them instantly in the browser without refreshing the entire page. That's the Hot Module Refresh from `@snowpack/plugin-svelte`.
+
+Image: GIF showing code and site side by side, site is a "Learn Svelte" link on a white background. When the text is edit to add "Hello world" and the file saves, the changes show up in the site immediately.
+
+> 💡 Tip: check out your network tab in the browser inspector- you'll see only the file that changed reloading, not all the files that make up the site.
```

Comment:
Ditto, strike this for now, not important to bring up here / this early in guide.